src/js/components/TextArea/doc.js

```diff
@@ -13,6 +13,9 @@ export const doc = TextArea => {
     .intrinsicElement('textarea');
 
   DocumentedTextArea.propTypes = {
+    a11yTitle: PropTypes.string.description(
+      'Custom title to be used by screen readers.',
```

Comment:
```suggestion
      'Custom label to be used by screen readers. When provided, an aria-label will be added to the element.',
```
A consistent message with other components.